azurerm/internal/services/web/registration.go

```diff
@@ -33,16 +33,17 @@ func (r Registration) SupportedDataSources() map[string]*schema.Resource {
 // SupportedResources returns the supported Resources supported by this Service
 func (r Registration) SupportedResources() map[string]*schema.Resource {
 	return map[string]*schema.Resource{
-		"azurerm_app_service_active_slot":                      resourceArmAppServiceActiveSlot(),
-		"azurerm_app_service_certificate":                      resourceArmAppServiceCertificate(),
-		"azurerm_app_service_certificate_order":                resourceArmAppServiceCertificateOrder(),
-		"azurerm_app_service_custom_hostname_binding":          resourceArmAppServiceCustomHostnameBinding(),
-		"azurerm_app_service_environment":                      resourceArmAppServiceEnvironment(),
-		"azurerm_app_service_plan":                             resourceArmAppServicePlan(),
-		"azurerm_app_service_slot":                             resourceArmAppServiceSlot(),
-		"azurerm_app_service_source_control_token":             resourceArmAppServiceSourceControlToken(),
-		"azurerm_app_service_virtual_network_swift_connection": resourceArmAppServiceVirtualNetworkSwiftConnection(),
-		"azurerm_app_service":                                  resourceArmAppService(),
-		"azurerm_function_app":                                 resourceArmFunctionApp(),
+		"azurerm_app_service_active_slot":                           resourceArmAppServiceActiveSlot(),
+		"azurerm_app_service_certificate":                           resourceArmAppServiceCertificate(),
+		"azurerm_app_service_certificate_order":                     resourceArmAppServiceCertificateOrder(),
+		"azurerm_app_service_custom_hostname_binding":               resourceArmAppServiceCustomHostnameBinding(),
+		"azurerm_app_service_environment":                           resourceArmAppServiceEnvironment(),
+		"azurerm_app_service_plan":                                  resourceArmAppServicePlan(),
+		"azurerm_app_service_slot":                                  resourceArmAppServiceSlot(),
+		"azurerm_app_service_source_control_token":                  resourceArmAppServiceSourceControlToken(),
+		"azurerm_app_service_virtual_network_swift_connection":      resourceArmAppServiceVirtualNetworkSwiftConnection(),
+		"azurerm_app_service_virtual_network_swift_connection_slot": resourceArmAppServiceVirtualNetworkSwiftConnectionSlot(),
```

Comment:
could we make this 
```suggestion
		"azurerm_app_service_slot_virtual_network_swift_connection": resourceArmAppServiceVirtualNetworkSwiftConnectionSlot(),
```

Reply:
I've named it this way is because when running the test just for the `azurerm_app_service_virtual_network_swift_connection` test, it would be impossible with the suggested naming, as both tests match.

Reply:
Should I rename it anyway?

Reply:
Oh. After reading this comment so many times I see that my argument was totally invalid and that you're even supporting it – kind of. A fix is incoming.